bluebottle/rewards/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from django_extensions.db.fields import (CreationDateTimeField,
+                                         ModificationDateTimeField)
+
+from bluebottle.utils.utils import StatusDefinition
+
+
+class Reward(models.Model):
+    """
+    Rewards for donations
+    """
+    amount = models.DecimalField(_('Amount'), max_digits=16, decimal_places=2)
+    title = models.CharField(_('Title'), max_length=30)
+    description = models.CharField(_('Description'), max_length=200)
+    project = models.ForeignKey('projects.Project', verbose_name=_('Project'))
+    limit = models.IntegerField(_('Limit'), null=True, blank=True,
+                                help_text=_('How many of this rewards are available'))
+
+    creation_date = CreationDateTimeField(_('creation date'))
```

Comment:
Our newer models seem to use created/updated for these fields names. Should we have a standard for this?